packages/contracts-bedrock/scripts/DeployImplementations.s.sol

```diff
@@ -37,7 +37,7 @@ import { DeployUtils } from "scripts/libraries/DeployUtils.sol";
 import { Solarray } from "scripts/libraries/Solarray.sol";
 
 // See DeploySuperchain.s.sol for detailed comments on the script architecture used here.
-contract DeployImplementationsInput {
+contract DeployImplementationsInput is Script {
```

Comment:
nit, we should have a shared default IO contract to inherit from, but in the meantime let's instead inherit from `CommonBase` like we do in DeploySuperchainInput. It just contains less stuff than `Script`, and is more clear since this contract isn't actually a script

Reply:
Added in: https://github.com/ethereum-optimism/optimism/pull/11875/commits/e230d9d3e044754183d41469a690cac0ff25b800